README.md

```diff
@@ -1,13 +1,13 @@
 # prebuild-install [![Build Status](https://travis-ci.org/mafintosh/prebuild-install.svg?branch=master)](https://travis-ci.org/mafintosh/prebuild-install)
 
-A command line tool for easily install prebuilds for multiple version of node/iojs on a specific platform.
+Command line tool to easily install prebuilds for multiple version of node/iojs
```

Comment:
Yes, but being the project description, it's not good being it too much long, it's better being a simple and short sentence with no wrapping.